lib/rules/no-irregular-whitespace/__tests__/index.mjs

```diff
@@ -113,6 +149,33 @@ testRule({
 			endLine: 1,
 			endColumn: 11,
 		},
+		{
+			code: '@media screen {\f}',
+			description: 'irregular whitespace in the after of an at rule',
+			message: messages.unexpected,
+			line: 1,
+			column: 16,
+			endLine: 1,
+			endColumn: 17,
+		},
+		{
+			code: '.a {\f}',
+			description: 'irregular whitespace in the after of an at rule',
+			message: messages.unexpected,
+			line: 1,
+			column: 5,
+			endLine: 1,
+			endColumn: 6,
+		},
+		{
+			code: '.a {\f\f}',
+			description: 'irregular whitespace in the after of an at rule',
```

Comment:
ditto at rule
whitespace > whitespaces

Reply:
`whitespace` is both plural and singular.
`whitespaces` isn't actually correct spelling, while `whitespace characters` is correct if you want to be explicit about the plural :)

I think `whitespace` is fine in this case.